resume-buddy/src/main/webapp/scripts/script.js

```diff
@@ -70,3 +73,4 @@ function populateCareers() {
       })
     });
 }
+
```

Comment:
Can you possibly auto-format this file to make sure indentations are 2 spaces?

https://google.github.io/styleguide/jsguide.html#formatting-block-indentation